src/comm/AcceptLimiter.cc

```diff
@@ -26,37 +26,32 @@ Comm::AcceptLimiter::defer(const Comm::TcpAcceptor::Pointer &afd)
 {
     ++ (afd->isLimited);
```

Comment:
Removed isLimited since it has no real purpose now, you can check if it is limited by looking to see if it exists in the set.

Reply:
I was just thinking that this member could be used as a flag now to avoid the search when adding. Does not matter enough to add back though.

Reply:
There should be no search when adding because the code should be written so that we never add twice (roughly speaking). If maintaining a boolean flag is the best way to maintain that invariant, we can convert this counter into such a flag, but I hope there are better ways to accomplish the same thing.